README.md

```diff
@@ -71,22 +75,47 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+"Fat Models, Skinny Controllers"
+
+Namely, any non-response-related logic should go in the model. The “skinny” controller is simply a nice interface between the view and model.
+
+sorry I know you said not to but why wouldn't you just?
+
+
+a<% if song.avg_rating < 5  %>
+    <td class="red"><%= song %></td>
+<% elsif song.avg_rating < 10 %>
+    <td class="yellow"><%= song %></td>
+<% else %>
+    <td class="green"><%= song %></td>
+<% end %>
+
+AND this was too confusing for me to figure out
+
```

Comment:
We haven't talked much about these, but the answer here is Helper Methods as these are intended to "help" render HTML based on conditions. Controller methods concern how data moves between routes. Model methods concern how data goes to and comes from the database.